pkg/mount/mountutil.go

```diff
@@ -116,11 +117,22 @@ func getDiskDevPath(partition string) (string, error) {
 		return "", err
 	}
 
-	parentDisk, ok := getParentBlockDevice(link)
-	if !ok {
-		return "", fmt.Errorf("could not find parent device for %s", link)
+	var disk string
```

Comment:
n_: Fix the function comment => `getDiskSysPath` to `getDiskDevPath` ==> Should this intead be called `getPartitionDevPath` ?

Reply:
Similar to other functions include information on how this function will be used. This will help in reviewing the code added under the feature flag.

Reply:
Done.